inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -185,6 +185,12 @@ cephfs_provisioner_enabled: false
 # cephfs_provisioner_secret: secret
 # cephfs_provisioner_storage_class: cephfs
 
+# Nginx ingress controller deployment
+nginx_ingress_enabled: false
+# nginx_ingress_namespace: "{{ system_namespace }}"
```

Comment:
by default nginx-ingress uses seperate namespace so why define system_namespace here? of course you can add in system namespace, but that's not proper way as it exposes all resources in system namespace for controller. Proposal is to make variable ingress_namespace and default value "ingress-nginx"

Reply:
I agree. We just tried the PR which works perfectly. But we changed the namespace to it's defalt "ingress-nginx"